contracts/src/instances/ezeth/EzETHTarget0.sol

```diff
@@ -30,6 +30,12 @@ contract EzETHTarget0 is HyperdriveTarget0, EzETHBase {
 
     /// Extras ///
 
+    /// @notice Returns the Renzo contract.
+    /// @return _restakeManager The Renzo contract.
+    function renzo() external view returns (IRestakeManager) {
```

Comment:
We should add ezETH too IMO. This would publicly display the immutable that is being used in the contract.

Reply:
Added ezETH